masonry/src/passes/event.rs

```diff
@@ -195,6 +195,23 @@ pub(crate) fn root_on_access_event(
         event,
         false,
         |widget, ctx, event| {
+            if event.target == ctx.widget_id() {
+                match event.action {
+                    accesskit::Action::Focus => {
+                        if ctx.is_in_focus_chain() && !ctx.is_disabled() && !ctx.is_focused() {
+                            ctx.request_focus();
```

Comment:
I think we should probably also call `ctx.set_handled()` here. See [#masonry>Missing event `set_handled` calls](https://xi.zulipchat.com/#narrow/stream/317477-masonry/topic/Missing.20event.20.60set_handled.60.20calls)